apps/cli-gui/src/app/app.component.html

```diff
@@ -1,14 +1,14 @@
-<ng-container *ngIf="core.coreState$ | async as coreState">
-  <header class="main-header">
-    <div class="project-selector">
-      <select name="project-selector" id="project-selector">
-        <option *ngFor="let project of coreState.projectNames" value="project">
-          {{ project }}
-        </option>
-      </select>
-    </div>
-  </header>
-  <main>
-    <router-outlet></router-outlet>
-  </main>
-</ng-container>
+<!--<ng-container *ngIf="core.coreState$ | async as coreState">-->
```

Comment:
if the comments are not needed or some playground @agroupp did I think you can remove them all together and start from scratch